reconcile/openshift_saas_deploy_trigger_base.py

```diff
@@ -351,6 +359,16 @@ def _construct_tekton_trigger_resource(
             "params": parameters,
         },
     }
+
+    if timeout:
+        if timeout < 60:
+            raise TektonTimeoutBadValueError(
+                f"timeout {timeout} is smaller than 60 minutes"
+            )
+
+        # conforming to Go’s ParseDuration format
+        body["spec"]["timeout"] = f"{timeout}m"
```

Comment:
would you consider using Go's ParseDuration format for the timeout already in the schema https://github.com/app-sre/qontract-schemas/pull/312

Reply:
It makes sense. It'll make the feature nicer.

Reply:
Fixed in cc63b993